ssc_interface_wrapper_ros2/src/converter_worker.cpp

```diff
@@ -68,19 +68,19 @@ namespace ssc_interface_wrapper{
         //Setup Subscribers
 
         //subscribers from CARMA
-        guidance_state_sub_ = create_subscription<carma_planning_msgs::msg::GuidanceState>("state", 1, 
+        guidance_state_sub_ = create_subscription<carma_planning_msgs::msg::GuidanceState>("state", 1,
                                                                     std::bind(&Converter::callback_from_guidance_state, this, std_ph::_1));
-        //subscribers from autoware                                                                  
+        //subscribers from autoware
         vehicle_cmd_sub_ = create_subscription<autoware_msgs::msg::VehicleCmd>("vehicle_cmd", 1,
                                                                     std::bind(&Converter::callback_from_vehicle_cmd, this, std_ph::_1));
         engage_sub_ = create_subscription<std_msgs::msg::Bool>("vehicle/engage", 1,
-                                                        std::bind(&Converter::callback_from_engage, this, std_ph::_1));                                                                    
-                                                                                                                                                             
+                                                        std::bind(&Converter::callback_from_engage, this, std_ph::_1));
+
         // ssc topic subscribers
-        module_states_sub_ = create_subscription<automotive_navigation_msgs::msg::ModuleState> ("as/module_states", 1, 
+        module_states_sub_ = create_subscription<automotive_navigation_msgs::msg::ModuleState> ("as/module_states", 1,
                                                                     std::bind(&Converter::callback_from_ssc_module_states, this, std_ph::_1));
         // TO DO: The output from callback_from_ssc_feedbacks and callback_for_twist_update needs to be kept synchronized
-        // This was being done in autoware.ai using message_filters https://github.com/usdot-fhwa-stol/autoware.ai/blob/1cb4e74b810111369a6bbe49b0e64e07767454c0/drivers/as/nodes/ssc_interface/ssc_interface.h#L58
+        // This was being done in autoware using message_filters https://github.com/usdot-fhwa-stol/autoware.ai/blob/1cb4e74b810111369a6bbe49b0e64e07767454c0/drivers/as/nodes/ssc_interface/ssc_interface.h#L58
```

Comment:
Message filters humble version should support synchronization using message_filters. This implementation should be  updated

Reply:
okay I removed the TODO comment

Reply:
For this one, we should be implementing the change suggested in the todo comments. We can probably just take the implementation from autoware which was updated here for foxy since message_filters wasn't supporting the required function on that release. https://github.com/usdot-fhwa-stol/autoware.ai/blob/1cb4e74b810111369a6bbe49b0e64e07767454c0/drivers/as/nodes/ssc_interface/ssc_interface.h#L58

Reply:
Ah I misunderstood your comment as I thought this was already being done. I ported them in.